application/modules/invoices/controllers/Invoices.php

```
@@ -285,19 +285,19 @@ public function generate_xml($invoice_id): void
         }
 
         // eInvoice library to Generate the appropriate UBL/CII or false
-        $xml_id    = $einvoice->name; // $invoice->client_einvoicing_version
-        $options   = [];
+        $xml_id = $einvoice->name; // $invoice->client_einvoicing_version
+        $options = [];
         $generator = $xml_id;
-        $path      = APPPATH . 'helpers/XMLconfigs/';
+        $path = APPPATH . 'helpers/XMLconfigs/';
         if ($xml_id && file_exists($path . $xml_id . '.php') && include $path . $xml_id . '.php') {
             $embed_xml = $xml_setting['embedXML'];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$embed_xml'. (undefined)

(UnusedLocalVariable)